src/util/setSelectionsAndFocusEditor.ts

```diff
@@ -78,21 +78,10 @@ function getViewColumn(editor: TextEditor): ViewColumn | undefined {
     return undefined;
   }
   const uri = editor.document.uri.toString();
-  const tabGroups: any[] = (window as any)?.tabGroups?.all ?? [];
-  const tabGroup = tabGroups.find(
-    (tabGroup: any) =>
-      !!tabGroup?.tabs.find(
-        (tab: any) => tab?.input?.modified?.toString() === uri
-      )
+  const tabGroup = (window as any)?.tabGroups?.all?.find((tabGroup: any) =>
```

Comment:
Do we need all these `?.`s?

Reply:
Probably not but since it's untyped I think it's better to be safe than sorry.